bundles/org.openhab.binding.energenie/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,141 @@
+				<label>Sockets</label>
+			</channel-group>
+		</channel-groups>
+
+		<config-description-ref uri="thing-type:energenie:config" />
+
+	</thing-type>
+
+	<thing-type id="pmslan">
+		<label>Energenie EG-PMS-LAN</label>
+		<description>Energenie programmable power strip with LAN interface</description>
+
+		<channel-groups>
+			<channel-group id="socketChannel" typeId="socketChannel">
+				<label>Sockets</label>
+			</channel-group>
+		</channel-groups>
+
+		<config-description-ref uri="thing-type:energenie:config" />
+
+	</thing-type>
+
+	<thing-type id="pms2lan">
+		<label>Energenie EG-PMS2-LAN</label>
+		<description>Energenie programmable power strip with LAN interface</description>
+
+		<channel-groups>
+			<channel-group id="socketChannel" typeId="socketChannel">
+				<label>Sockets</label>
+			</channel-group>
+		</channel-groups>
+
+		<config-description-ref uri="thing-type:energenie:config" />
+
+	</thing-type>
+
+	<thing-type id="pmswlan">
+		<label>Energenie EG-PMS-WLAN</label>
+		<description>Energenie programmable power strip with WLAN interface</description>
+
+		<channel-groups>
+			<channel-group id="socketChannel" typeId="socketChannel">
+				<label>Sockets</label>
+			</channel-group>
+		</channel-groups>
+
+		<config-description-ref uri="thing-type:energenie:config" />
+
+	</thing-type>
+
+	<thing-type id="pwmlan">
+		<label>Energenie EGM-PWM-LAN</label>
+		<description>Energenie energy meter with LAN interface</description>
+
+		<channels>
+			<channel id="voltage" typeId="voltage" />
+			<channel id="current" typeId="current" />
+			<channel id="power" typeId="power" />
+			<channel id="energy" typeId="energy" />
+		</channels>
+
+		<config-description-ref uri="thing-type:energenie:config" />
+
+	</thing-type>
+
+	<channel-group-type id="socketChannel">
+		<label>Socket channel</label>
+		<description>A Energenie relay channel</description>
+		<channels>
+			<channel id="socket1" typeId="socket1" />
+			<channel id="socket2" typeId="socket2" />
+			<channel id="socket3" typeId="socket3" />
+			<channel id="socket4" typeId="socket4" />
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="socket1">
+		<item-type>Switch</item-type>
+		<label>Socket 1 state</label>
+		<description>Channel representing the state of socket 1</description>
+	</channel-type>
+
+	<channel-type id="socket2">
+		<item-type>Switch</item-type>
+		<label>Socket 2 state</label>
+		<description>Channel representing the state of socket 2</description>
+	</channel-type>
+
+	<channel-type id="socket3">
+		<item-type>Switch</item-type>
+		<label>Socket 3 state</label>
+		<description>Channel representing the state of socket 3</description>
+	</channel-type>
+
+	<channel-type id="socket4">
+		<item-type>Switch</item-type>
+		<label>Socket 4 state</label>
+		<description>Channel representing the state of socket 4</description>
+	</channel-type>
+
+	<channel-type id="voltage">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Voltage </label>

```

Comment:
All the Number labels have an extra space:
```suggestion
		<label>Voltage</label>
```

Reply:
Done